client/src/modules/Globals/Navbar.tsx

```diff
@@ -68,12 +68,12 @@ export default function Navbar({ userInput }: NavbarProps) {
         <a className="" href="/">
           <img
             src="/logo.svg"
-            className="img-fluid scale-logo-navbar"
+            className="scale-logo-navbar"
             alt="CU Reviews Logo"
           />
         </a>
       </div>
-      <div className="col navbar-searchbar-container">
+      <div className={`col navbar-searchbar-container${isLoggedIn ? "-wider" : "-not"}`}>
```

Comment:
More so for convention, instead of `-not` , you can just leave the `className` as an empty string if the user is not logged in. So, if `isLoggedIn` is `true`, then the `className` will be `navbar-searchbar-container`, but is it's `false`, then the `className` will be `navbar-searchbar-container`. Also,

nit: can we change the `className` from `navbar-searchbar-container-wider` to `navbar-searchbar-container-profile`, or something more related to why this `className` is being used?